File: ployglots/mta-ployglot-pkg-figma-api/packages/ts/src/resources/me.ts
// @ts-nocheck
import type { FigmaClient } from '../figma-client.js';
import type { FigmaUser } from './types.js';

export class MeResource {
  constructor(private readonly client: FigmaClient) {}

  /** GET /v1/me — authenticated user. */
  async get(): Promise<FigmaUser> {
    return this.client.get<FigmaUser>('/v1/me');
  }
}

File: ployglots/mta-ployglot-pkg-figma-api/packages/py/figma_api/resources/me.py
from __future__ import annotations

from typing import TYPE_CHECKING, Any

if TYPE_CHECKING:
    from .._figma_client import FigmaClient


class MeResource:
    def __init__(self, client: FigmaClient) -> None:
        self._client = client

    async def get(self) -> dict[str, Any]:
        """GET /v1/me — authenticated user."""
        return await self._client.get("/v1/me")
